### PR TITLE
Creates protobuf for VID model schema, and necessary protobuf placeholder for event schema.

### PR DESCRIPTION
**Pull request by tcsnfkx**

### CODE REVIEW

File: src/proto/model.proto [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+  // person, and other associated attributes.
+  // This is repeated because we allow multiple VIDs for single input event.
+  // For each event, there will always be 1 or more "virtual_person_activity"s.
+  // For each "virtual_person_activity", if virtual_person_id is not set, it is
+  // only for counting impressions by labels, and there is no virtual person for
+  // reach purpose.
+  repeated VirtualPersonActivity virtual_person_activities = 2;
+
+  // Sets to true if no user id is set for this event.
+  optional bool is_user_id_free_person = 3;
```
Comment (lenjoy) [inline src/proto/model.proto:252]: input or output?  which field is for user_id?

File: src/proto/model.proto [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+  // Country the person is from.
+  optional string person_country_code = 4;
+
+  // Region within a country the person is from.
+  optional string person_region_code = 5;
```
Comment (lenjoy) [inline src/proto/model.proto:258]: which country/region code are we going to use?

File: src/proto/event.proto [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package virtual_people;
+
+// Placeholder for DemoBucket message.
+message DemoBucket {}
```
Comment (lenjoy) [inline src/proto/event.proto:20]: any age/gender proposal yet?

File: src/proto/event.proto [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+// Placeholder for LabelerInput message.
+message LabelerInput {}
+
+// Placeholder for VirtualPersonActivity message.
+message VirtualPersonActivity {}
```
Comment (lenjoy) [inline src/proto/event.proto:26]: `virtual person` vs `virtual people`, have we finalized the name yet?

File: src/proto/model.proto [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+  //   total_population = 100
+  // The pool covers from 1000 to 1099.
+  message VirtualPersonPool {
+    optional uint64 population_offset = 1;
+    optional uint64 total_population = 2;
+  }
+
+  repeated VirtualPersonPool pools = 1;
+
+  optional string random_seed = 2;
```
Comment (lenjoy) [inline src/proto/model.proto:67]: is rand seed string or int?

File: AUTHORS [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# This is the list of Virtual People ID Systems' significant contributors.
+# This is the list of Cross-Media Measurement's significant contributors.
```
Comment (lenjoy) [inline AUTHORS:1]: isn't this virtual people ID repo?

File: src/proto/model.proto (hunk 1 above)
Comment (tcsnfkx) [reply src/proto/model.proto:252]: This field is used during applying the model, which is neither input nor output.

user_id is a field in the event schema, and will be added to event.proto in a future PR.

File: src/proto/model.proto (hunk 5 above)
Comment (tcsnfkx) [reply src/proto/model.proto:67]: It is string.

File: src/proto/model.proto (hunk 2 above)
Comment (tcsnfkx) [reply src/proto/model.proto:258]: This is the structure which represents a region in the model. From the perspective of model schema, there is no restriction about the value of these fields. In a specific model, the allowed values of country/region code depend on the supported regions of that model.

File: src/proto/event.proto (hunk 4 above)
Comment (tcsnfkx) [reply src/proto/event.proto:26]: "virtual person" represents a single "virtual person", while "virtual people" refers to a set of "virtual person"s.

File: src/proto/event.proto (hunk 3 above)
Comment (tcsnfkx) [reply src/proto/event.proto:20]: The details will be added in a future PR.

File: AUTHORS (hunk 6 above)
Comment (tcsnfkx) [reply AUTHORS:1]: Virtual People ID Systems is part of Cross Media Measurement project. We want to keep this aligned with other Cross Media Measurement repos.

**Review by lenjoy — APPROVED** [review]

**Review by EvgSkv — APPROVED** [review]

File: src/proto/model.proto [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+      SparseUpdateMatrix sparse_update_matrix = 2;
+      ConditionalMerge conditional_merge = 3;
+      UpdateTree update_tree = 4;
+    }
+    // The random seed used for the updater.
+    optional string random_seed = 5;
+  }
+
+  message AttributesUpdaters {
+    repeated AttributesUpdater updates = 2;
```
Comment (wliue) [inline src/proto/model.proto:120]: Should this be 1?

Comment (tcsnfkx) [reply src/proto/model.proto]: Done.

**Review by wliue — APPROVED** [review]

File: src/proto/event.proto (hunk 4 above)
Comment (oliver-amzn) [reply src/proto/event.proto:26]: Might be a useful piece of information to add somewhere, in a README or in the proto files themselves.

**Review by oliver-amzn — APPROVED** [review]

File: src/proto/event.proto (hunk 4 above)
Comment (tcsnfkx) [reply src/proto/event.proto:26]: The explanation was added in the comments of virtual_person_activities in model.proto.

**Pull request merged**